pkg/crc/api/client/client.go

```diff
@@ -109,17 +110,8 @@ func (c *Client) WebconsoleURL() (ConsoleResult, error) {
 
 func (c *Client) GetConfig(configs []string) (GetConfigResult, error) {
 	var gcr = GetConfigResult{}
-	var data = new(bytes.Buffer)
-
-	if len(configs) > 0 {
-		cfg := GetOrUnsetConfigRequest{
-			Properties: configs,
-		}
-		if err := json.NewEncoder(data).Encode(cfg); err != nil {
-			return gcr, fmt.Errorf("Failed to encode data to JSON: %w", err)
-		}
-	}
-	body, err := c.sendPostRequest("/config/get", data)
+	queryString := strings.Join(configs, "&")
```

Comment:
@cfergeau didn't get you why I need to escape individual config, here we are building the query string using the `&` as separator? This should be parsed by handler like https://pkg.go.dev/net/url#URL.Query way (example)

Reply:
If one calls `client.GetConfig([]string{ "a&a", "b&b", "c&c"})`, the current code generates `queryString = "a&a&b&b&c&c"`. After the call to `strings.Join`, we've lost the individual query item, so there's nothing golang URL module can do about it. If you call `url.QueryEscape` beforehand, you can generate the expected `queryString = "a%26a&b%26b&c%26c"`, which `URL.Query` will then be able to handle properly.

Reply:
(this is all very theoretical as our config options don't have weird characters in them, but since we don't control the inputs, and it's not very hard to do the right thing, I think it's worthwhile to do the right thing)